jupyter-on-gke/allowlist

```diff
@@ -1,3 +1,2 @@
-user:example@google.com
-user:example2@google.com
-group:example-groupd@google.com
\ No newline at end of file
+allUsers
+user:umeshkumhar@google.com
```

Comment:
revert this?

Reply:
same here

Reply:
sorry, lost the original comment. should we remove your ldap?